packages/webservice/app/src/main/resources/models/tables/queryStats.hjson

```diff
@@ -0,0 +1,113 @@
+{
+    tables:
+    [
+        {
+            name: QueryStats
+            table: QueryStats
+            description:
+            '''
+            Carbon Query Stats Table
+            '''
+            cardinality: large
+            measures :
+            [
+                {
+                    name: durationInMillis
+                    type: DECIMAL
+                    definition: IF (durationInMillis IS NULL, -1, durationInMillis)
```

Comment:
Check out the latest documentation for HJSON configuration:
https://elide.io/pages/guide/v5/04-analytics.html

Any column referenced in the definition should be wrapped in {{ and }}.  So definition here should be:
`IF ({{durationInMillis}} IS NULL, -1, {{durationInMillis}})`